### PR TITLE
Fix inconsistent wording

### PR DESCRIPTION
**Pull request by hlts2**
### Description:

I fixed inconsistent wording about kubernetes and Go.
For example, I change from k8s to Kubernetes and golang to Go.

### Environment:

- Golang Version: 1.14.3
- Docker Version: 19.03.8
- Kubernetes Version: 1.18.2
- NGT Version: 1.11.5

### Types of changes:

- [ ] Bug fix [type/bug]
- [ ] New feature [type/feature]
- [ ] Add tests [type/test]
- [ ] Security related changes [type/security]
- [x] Add documents [type/documentation]
- [ ] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [ ] Update CI [type/ci]

### Changes to Core Features:

- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### Checklist:

- [x] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [ ] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [ ] I have added tests and benchmarks to cover my changes.
- [ ] I have ensured all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by kpango** [comment]
could you please update go version and k8s version about each template?

**Comment by rinx** [comment]
@hlts2 Please update the workflow to work correctly.
https://github.com/vdaas/vald/blob/fc9995ac620115157838a7186adfc3fff3ded85d/.github/workflows/update_pull_request_template.yml#L29

I recommend to add sed commands to update issue template same time.

**Comment by kevindiu** [comment]
> @hlts2 Please update the workflow to work correctly.
> https://github.com/vdaas/vald/blob/fc9995ac620115157838a7186adfc3fff3ded85d/.github/workflows/update_pull_request_template.yml#L29
> 
> I recommend to add sed commands to update issue template at the same time.

should we include this changes in this PR?

File: .github/workflows/update_pull_request_and_issue_template.yml [hunk 1]
```diff
@@ -19,21 +19,26 @@ jobs:
...
           TIMESTAMP=$(date +%Y%m%d_%H%M%S_%3N)
-          BRANCH_NAME="documentation/pull_request_template/update_versions_${TIMESTAMP}"
+          BRANCH_NAME="documentation/pull_request_and_issue_template/update_versions_${TIMESTAMP}"
           git checkout master
           git checkout -b ${BRANCH_NAME}
 
           GO_VERSION=$(make version/go)
           NGT_VERSION=$(make version/ngt)
 
+          sed -i -e "s/^- Golang Version: .*$/- Golang Version: ${GO_VERSION}/" .github/ISSUE_TEMPLATE/bug_report.md
```
Comment (kevindiu) [inline .github/workflows/update_pull_request_and_issue_template.yml:29]:
```suggestion
          sed -i -e "s/^- Go Version: .*$/- Go Version: ${GO_VERSION}/" .github/ISSUE_TEMPLATE/bug_report.md
```

File: .github/workflows/update_pull_request_and_issue_template.yml [hunk 2]
```diff
@@ -19,21 +19,26 @@ jobs:
...
+          sed -i -e "s/^- Golang Version: .*$/- Golang Version: ${GO_VERSION}/" .github/ISSUE_TEMPLATE/security_issue_report.md
```
Comment (kevindiu) [inline .github/workflows/update_pull_request_and_issue_template.yml:30]:
```suggestion
          sed -i -e "s/^- Go Version: .*$/- Golang Version: ${GO_VERSION}/" .github/ISSUE_TEMPLATE/security_issue_report.md
```

File: .github/workflows/update_pull_request_and_issue_template.yml [hunk 3]
```diff
@@ -19,21 +19,26 @@ jobs:
...
           sed -i -e "s/^- Golang Version: .*$/- Golang Version: ${GO_VERSION}/" .github/PULL_REQUEST_TEMPLATE.md
```
Comment (kevindiu) [inline .github/workflows/update_pull_request_and_issue_template.yml:31]:
```suggestion
          sed -i -e "s/^- Go Version: .*$/- Go Version: ${GO_VERSION}/" .github/PULL_REQUEST_TEMPLATE.md
```

File: .github/workflows/update_pull_request_and_issue_template.yml [hunk 4]
```diff
@@ -1,4 +1,4 @@
-name: 'Update PULL_REQUEST_TEMPLATE'
+name: 'Update PULL_REQUEST_TEMPLATE and ISSUE_TEMPLATE'
```
Comment (kevindiu) [inline .github/workflows/update_pull_request_and_issue_template.yml:1]: do we need to update anything if we update this file path?

File: .github/workflows/update_pull_request_and_issue_template.yml (hunk 1 above)
Comment (hlts2) [reply .github/workflows/update_pull_request_and_issue_template.yml:29]: sorry. I will fix it soon

File: charts/vald/README.md [hunk 5]
```diff
@@ -845,7 +845,7 @@ Please run the following command to install the chart with this values yaml,
     $ helm repo add vald https://vald.vdaas.org/charts
     $ helm install --values values-agent-ngt-standalone.yaml vald-agent-ngt vald/vald
 
-If you'd like to access the agents from out of the Kubernetes cluster, it is recommended to create an [Ingress][k8s-ingress].
+If you'd like to access the agents from out of the Kubernetes cluster, it is recommended to create an [Ingress][kubernetes-ingress].
```
Comment (vankichi) [inline charts/vald/README.md:848]: 😇

File: charts/vald/README.md.gotmpl [hunk 6]
```diff
@@ -63,7 +63,7 @@ Please run the following command to install the chart with this values yaml,
     $ helm repo add vald https://vald.vdaas.org/charts
     $ helm install --values values-agent-ngt-standalone.yaml vald-agent-ngt vald/vald
 
-If you'd like to access the agents from out of the Kubernetes cluster, it is recommended to create an [Ingress][k8s-ingress].
+If you'd like to access the agents from out of the Kubernetes cluster, it is recommended to create an [Ingress][kubernetes-ingress].
```
Comment (vankichi) [inline charts/vald/README.md.gotmpl:66]: 😇

File: docs/developer/style.md [hunk 7]
```diff
@@ -24,13 +24,13 @@
 
 ## Introduction
 
-This guideline includes the coding style for all Vald contributors and reviewers. Everyone should follow this guideline to keep the style consistent so everyone can understand and contribute to Vald easier once they learn this guideline. You should have the basic knowledge of how to write Golang before contributing to Vald. If you found any bug please create [a GitHub issue](https://github.com/vdaas/vald/issues/new?assignees=&labels=type%2Fbug%2C+priority%2Fmedium%2C+team%2Fcore&template=bug_report.md&title=) and we will work on it.
+This guideline includes the coding style for all Vald contributors and reviewers. Everyone should follow this guideline to keep the style consistent so everyone can understand and contribute to Vald easier once they learn this guideline. You should have the basic knowledge of how to write Go language before contributing to Vald. If you found any bug please create [a GitHub issue](https://github.com/vdaas/vald/issues/new?assignees=&labels=type%2Fbug%2C+priority%2Fmedium%2C+team%2Fcore&template=bug_report.md&title=) and we will work on it.
```
Comment (vankichi) [inline docs/developer/style.md:27]:
```suggestion
This guideline includes the coding style for all Vald contributors and reviewers. Everyone should follow this guideline to keep the style consistent so everyone can understand and contribute to Vald easier once they learn this guideline. You should have the basic knowledge of how to write Go before contributing to Vald. If you found any bug please create [a GitHub issue](https://github.com/vdaas/vald/issues/new?assignees=&labels=type%2Fbug%2C+priority%2Fmedium%2C+team%2Fcore&template=bug_report.md&title=) and we will work on it.
```

File: charts/vald/README.md (hunk 5 above)
Comment (hlts2) [reply charts/vald/README.md:848]: thank you for your review.
I will fix it soon.

File: charts/vald-helm-operator/README.md [hunk 8]
```diff
@@ -69,7 +69,7 @@ Configuration
 | rbac.name | string | `"vald-helm-operator"` | name of roles and rolebindings |
 | reconcilePeriod | string | `"1m"` | reconcile duration of operator |
 | replicas | int | `2` | number of replicas |
-| resources | object | `{}` | k8s resources of pod |
+| resources | object | `{}` | kubernetes resources of pod |
```
Comment (rinx) [inline charts/vald-helm-operator/README.md:72]: If you'd like to revise here, it is recommended to review values.yaml in helm operator.
This markdown is automatically generated from it.

Comment (hlts2) [reply charts/vald-helm-operator/README.md:72]: Thank you your feedback.
I fixed comment of values.yaml.

**Pull request merged**